### PR TITLE
Revised the AllowManagementOS parameter description

### PR DESCRIPTION
**Pull request by ManikaDhiman**

### CODE REVIEW

**Comment by ManikaDhiman** [comment]
#sign-off

File: docset/winserver2012-ps/hyper-v/Set-VMSwitch.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ This example sets the minimum bandwidth allocation to 500Mbps on a virtual switc
 ## PARAMETERS
 
 ### -AllowManagementOS
-Specifies whether the management operating system can use the physical network adapter that is bound to the external virtual switch.
+Specifies whether the management operating system can use the physical network adapter that is bound to the virtual switch. Setting AllowManagementOS to true creates management operating system virtual network adapters (VMNetworkAdapters) in the parent partition and connects it to the virtual switch. Setting AllowManagementOS to false removes any management operating system virtual network adapters (VMNetworkAdapters) connected to the virtual switch.
```
Comment (sdwheeler) [inline docset/winserver2012-ps/hyper-v/Set-VMSwitch.md:85]:
```suggestion
Specifies whether the management operating system can use the physical network adapter that is bound to the virtual switch. Setting **AllowManagementOS** to `$true` creates management operating system virtual network adapters (VMNetworkAdapters) in the parent partition and connects it to the virtual switch. Setting **AllowManagementOS** to `$false` removes any VMNetworkAdapters connected to the virtual switch.
```

File: docset/winserver2012r2-ps/hyper-v/Set-VMSwitch.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ This example sets the minimum bandwidth allocation to 500Mbps on a virtual switc
 ## PARAMETERS
 
 ### -AllowManagementOS
-Specifies whether the management operating system can use the physical network adapter that is bound to the external virtual switch.
+Specifies whether the management operating system can use the physical network adapter that is bound to the virtual switch. Setting AllowManagementOS to true creates management operating system virtual network adapters (VMNetworkAdapters) in the parent partition and connects it to the virtual switch. Setting AllowManagementOS to false removes any management operating system virtual network adapters (VMNetworkAdapters) connected to the virtual switch.
```
Comment (sdwheeler) [inline docset/winserver2012r2-ps/hyper-v/Set-VMSwitch.md:87]:
```suggestion
Specifies whether the management operating system can use the physical network adapter that is bound to the virtual switch. Setting **AllowManagementOS** to `$true` creates management operating system virtual network adapters (VMNetworkAdapters) in the parent partition and connects it to the virtual switch. Setting **AllowManagementOS** to `$false` removes any VMNetworkAdapters connected to the virtual switch.
```

File: docset/winserver2016-ps/hyper-v/Set-VMSwitch.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ This example sets the minimum bandwidth allocation to 500Mbps on a virtual switc
 ## PARAMETERS
 
 ### -AllowManagementOS
-Specifies whether the management operating system can use the physical network adapter that is bound to the external virtual switch.
+Specifies whether the management operating system can use the physical network adapter that is bound to the virtual switch. Setting AllowManagementOS to true creates management operating system virtual network adapters (VMNetworkAdapters) in the parent partition and connects it to the virtual switch. Setting AllowManagementOS to false removes any management operating system virtual network adapters (VMNetworkAdapters) connected to the virtual switch.
```
Comment (sdwheeler) [inline docset/winserver2016-ps/hyper-v/Set-VMSwitch.md:107]:
```suggestion
Specifies whether the management operating system can use the physical network adapter that is bound to the virtual switch. Setting **AllowManagementOS** to `$true` creates management operating system virtual network adapters (VMNetworkAdapters) in the parent partition and connects it to the virtual switch. Setting **AllowManagementOS** to `$false` removes any VMNetworkAdapters connected to the virtual switch.
```

File: docset/winserver2019-ps/hyper-v/Set-VMSwitch.md [hunk 4]
```diff
@@ -104,7 +104,7 @@ This example sets the minimum bandwidth allocation to 500Mbps on a virtual switc
 ## PARAMETERS
 
 ### -AllowManagementOS
-Specifies whether the management operating system can use the physical network adapter that is bound to the external virtual switch.
+Specifies whether the management operating system can use the physical network adapter that is bound to the virtual switch. Setting AllowManagementOS to true creates management operating system virtual network adapters (VMNetworkAdapters) in the parent partition and connects it to the virtual switch. Setting AllowManagementOS to false removes any management operating system virtual network adapters (VMNetworkAdapters) connected to the virtual switch.
```
Comment (sdwheeler) [inline docset/winserver2019-ps/hyper-v/Set-VMSwitch.md:107]:
```suggestion
Specifies whether the management operating system can use the physical network adapter that is bound to the virtual switch. Setting **AllowManagementOS** to `$true` creates management operating system virtual network adapters (VMNetworkAdapters) in the parent partition and connects it to the virtual switch. Setting **AllowManagementOS** to `$false` removes any VMNetworkAdapters connected to the virtual switch.
```

File: docset/winserver2022-ps/hyper-v/Set-VMSwitch.md [hunk 5]
```diff
@@ -104,7 +104,7 @@ This example sets the minimum bandwidth allocation to 500Mbps on a virtual switc
 ## PARAMETERS
 
 ### -AllowManagementOS
-Specifies whether the management operating system can use the physical network adapter that is bound to the external virtual switch.
+Specifies whether the management operating system can use the physical network adapter that is bound to the virtual switch. Setting AllowManagementOS to true creates management operating system virtual network adapters (VMNetworkAdapters) in the parent partition and connects it to the virtual switch. Setting AllowManagementOS to false removes any management operating system virtual network adapters (VMNetworkAdapters) connected to the virtual switch.
```
Comment (sdwheeler) [inline docset/winserver2022-ps/hyper-v/Set-VMSwitch.md:107]:
```suggestion
Specifies whether the management operating system can use the physical network adapter that is bound to the virtual switch. Setting **AllowManagementOS** to `$true` creates management operating system virtual network adapters (VMNetworkAdapters) in the parent partition and connects it to the virtual switch. Setting **AllowManagementOS** to `$false` removes any VMNetworkAdapters connected to the virtual switch.
```

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**